src/docs/go-grpc.yaml

```diff
@@ -0,0 +1,19 @@
+slug: go-grpc
+name: Go gRPC
+written: September 14, 2021
+updated: September 14, 2021
+protocol: gRPC
+port: 443
+# paths:
+#   serverCert: /etc/ssl/server.crt
+#   serverKey: /etc/ssl/server.key
+#   rootCert: /etc/ssl/root_ca.crt
```

Comment:
These three will probably be needed to sub into systemd / linux section templates. Make sure they match what you have in your server configuration. If you don't have explicit paths, you can just use `/path/to/server.crt` both here and in your server code.

Reply:
Fixed in fc2b7c2b55deeb771d0ea6c7cf076a07e93015fa